apps/examples/testcase/ta_tc/systemio/itc/itc_uart.c

```diff
@@ -62,6 +60,39 @@ void itc_iotbus_uart_init_stop_p(void)
 }
 
 /**
+* @testcase         itc_iotbus_uart_init_stop_multi_handle_p
+* @brief            initializes and closes uart_context
+* @scenario         initializes and closes uart_context
+* @apicovered       iotbus_uart_init, iotbus_uart_stop
+* @precondition     none
+* @postcondition    none
+*/
+void itc_iotbus_uart_init_stop_multi_handle_p(void)
+{
+	int ret = IOTBUS_ERROR_NONE;
+
+	iotbus_uart_context_h h_uart1 = iotbus_uart_init(DEVPATH);
+	TC_ASSERT_NEQ("iotbus_uart_init1", h_uart1, NULL);
+
+	iotbus_uart_context_h h_uart2 = iotbus_uart_init(DEVPATH);
+	TC_ASSERT_NEQ_CLEANUP("iotbus_uart_init2", h_uart2, NULL, iotbus_uart_stop(h_uart1));
+
+	iotbus_uart_context_h h_uart3 = iotbus_uart_init(DEVPATH);
+	TC_ASSERT_NEQ_CLEANUP("iotbus_uart_init3", h_uart3, NULL, iotbus_uart_stop(h_uart2); iotbus_uart_stop(h_uart3));
```

Comment:
We should stop 1 and 2 in failure. right?

Reply:
@sunghan-chang  
Yes , fixed.